src/inc/til/point.h

```diff
@@ -140,6 +140,50 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             return point{ x, y };
         }
 
+        point operator*(const int& scale) const

```

Comment:
Nnnnn should this be a template the same way that `CheckMul` is a template so it can accept any basic type?

Reply:
I was seriously just thinking about that. I'm not positive I'm skilled enough in template-fu but I'll give it a shot

Reply:
I'm usually not **that** skilled in template-fu. In this case, you could probably steal some of the template declaration from `CheckMul` itself by looking how it is defined.